src/snap/flags.cc

```diff
@@ -36,6 +38,17 @@ limitations under the License.
 
 namespace po = boost::program_options;
 namespace spd = spdlog;
+using json = nlohmann::json;
+
+void Plugin::Flags::setupPrettyFlags() {
+  _pretty_flags["TLSEnabled"] = "tls";
```

Comment:
Convert `_pretty_flags` to class variable, at it doesn't contain any instance-specific data.